packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -212,7 +212,7 @@ deployWebApiFunctionApp() {
     deployWebApiArmTemplate $packageName
 
     # Add reply url to function app registration
-    if [ $environment = "dev" ]; then
+    if [ "$clientIdPassed" = false]; then
```

Comment:
Do we need double quotes around var here?

Reply:
Yes, if the variable is not defined (which it isn't unless no client id is given) then it would fail without the quotes